android/src/main/java/cl/json/ShareFile.java

```diff
@@ -139,7 +134,9 @@ public Uri getURI() {
             }
         } else if(this.isLocalFile()) {
             Uri uri = Uri.parse(this.url);
-
+            if (uri.getPath() == null) {
+                return null;
```

Comment:
is this a breaking change?

Reply:
A good question - no intention to break - the suggestion here was part of Android Studio lint, it notified that uri.getPath() *may* be null, so I put a check in. I wasn't sure what to do with it though, what would the consequence of proceeding with previous code if uri.getPath() had been null? I think it would have thrown RuntimeException. I suppose we could null check farther up the call chain, or throw RuntimeException ourselves with a good message, or let it continue as previous (without this change) where null may infiltrate there

Reply:
Makes sense.
I think we can go on this way and test behaviours. If `return null` does not work, we could try to throw a `RuntimeException`, like your suggestion.
We can also try to solve this on JS land.